.github/ISSUE_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Pull_request
+about: Describe the issue related to pull requests here.
+title: ''
+labels: ''
+assignees: ''
+
+---
```

Comment:
Also, add the following question here.
'''
Use one of the following below:
1. Address  #(issue no) - use this if your PR partially solves the issue
2. Fixes #(issue no) - use this if your PR completely solves the issue
'''